clock/app/javascript/packs/application.js

```diff
@@ -19,11 +21,10 @@ import $ from 'jquery';
 //= require popper
 //= require bootstrapn
 
-
-
 $(document).ready(function() {
   //$("#store-errors").hide();
   $(".close").click(function() {
     $("#store-errors").toggle("slow");
   });
```

Comment:
you don't need Jquery here, you can refactor this code like this:
```
document.addEventListener("DOMContentLoaded",  function(event) { 
  var element = document.getElementById("store-error");
  element.classList.toggle("slow");
});
```

Reply:
Gracias, estaba buscando como convertir ese jquery a vanilla